### PR TITLE
chore: use kres to manage project

### PR DESCRIPTION
**Pull request by frezbo**
Move to using kres to manage project.

### CODE REVIEW

File: storage/iscsi-tools/open-iscsi/pkg.yaml [hunk 1]
```diff
@@ -33,23 +34,27 @@ steps:
...
+
+        pip3 install ninja
     build:
       - |
         export PKG_CONFIG_PATH=/usr/lib/pkgconfig
 
-        # musl doesn't have glob_onlydir but by spec it's also not guaranteed to return only dirs anyway
+        mkdir -p output
+
         LDFLAGS="$LDFLAGS -L/usr/local/lib" \
```
Comment (frezbo) [inline storage/iscsi-tools/open-iscsi/pkg.yaml:45]: the `make` way of building was already deprecated for `iscsi-tools` for a while, bringing in gcc 13 broke the build, so switch to meson

**Review by smira — APPROVED** [review]

**Comment by smira** [comment]
do we have reproducibility test as well?

**Comment by frezbo** [comment]
> do we have reproducibility test as well?

we never had for extensions, should I add one?

**Comment by smira** [comment]
> > do we have reproducibility test as well?
> 
> we never had for extensions, should I add one?

might be great to have it... less of an issue now, but we should be reproducible?

**Comment by frezbo** [comment]
> > > do we have reproducibility test as well?
> > 
> > 
> > we never had for extensions, should I add one?
> 
> might be great to have it... less of an issue now, but we should be reproducible?

added reproducibility too, `SOURCE_DATE_EPOCH` was missing for some

**Pull request merged**